diagnostic.md

```diff
@@ -73,14 +88,27 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request POST http://www.example.com/donuts/ \
+--header "Content-Type: application/json" \
+--data '{
+  "donut": {
+    "name": "French Cruller"",
+    "price": "$0.99"
+  }
+}'
 ```
 
 Write an AJAX request to create a single donut on the server using JSON. Please
 do not use `data = getFormFields(form)` instead write out the data object.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function (data) {
+  return $.ajax({
+    url: config.apiOrigin + '/donuts',
+    method: 'POST',
+    data,
+  })
+};
```

Comment:
We were looking for hard coded data here as well. This will only work with a click handler set up properly to hand it the right data. 

```js
let createDonut = function () {
  return $.ajax({
    url: 'http://example.com/donuts',
    method: 'POST',
    data: {
      "donut" {
        "name": "French Cruller",
        "price": "$0.99"
      }
    }
  });
};
```